release.nix

```diff
@@ -5,6 +5,12 @@
   changelog = {
     # Find the current version number with `git log --pretty=%h | wc -l`
     entries = [
+      {
+        version = 429;
+        changes = ''
+          - Re-introduced `lorri shell` command.
```

Comment:
Would add a reasoning here so users see it on update.

Reply:
Let's delete this entry until #297 is done.

Reply:
Done